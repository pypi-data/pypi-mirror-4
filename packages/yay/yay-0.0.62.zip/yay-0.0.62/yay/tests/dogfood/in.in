
list:
  - dog
  - cat
  - badger

list2:
  - dog
  - mouse

res1.foreach p in list2 if p in list: ${p}

