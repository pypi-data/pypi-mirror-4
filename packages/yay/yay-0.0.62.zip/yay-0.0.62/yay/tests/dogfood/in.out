
list:
  - dog
  - cat
  - badger

list2:
  - dog
  - mouse

res1:
  - dog

